dashboard/src/components/InfoCard/InfoCard.tsx

```diff
@@ -10,46 +10,50 @@ export interface IServiceInstanceCardProps {
   info: string;
   link?: string;
   icon?: string;
+  description?: string | JSX.Element;
   tag1Class?: string;
   tag1Content?: string | JSX.Element;
   tag2Class?: string;
   tag2Content?: string | JSX.Element;
 }
 
 const InfoCard: React.SFC<IServiceInstanceCardProps> = props => {
-  const { title, link, info, tag1Content, tag1Class, tag2Content, tag2Class } = props;
+  const { title, link, info, description, tag1Content, tag1Class, tag2Content, tag2Class } = props;
   const icon = props.icon ? props.icon : placeholder;
   return (
     <Card responsive={true} className="ListItem">
       <Link to={link || "#"} title={title}>
         <CardIcon icon={icon} />
-        <CardContent>
-          <div className="ListItem__content">
+      </Link>
```

Comment:
Now the link is just for the icon and the title, the description doesn't contain a link.

Reply:
Wouldn't make sense to make the whole thing linkable? Maybe wrapping Card inside the Link.

Reply:
that's how it was before, but I found weird that some piece of text (the description) is also a link:

<img width="322" alt="screenshot 2018-12-17 at 10 10 04" src="https://user-images.githubusercontent.com/4025665/50077197-0041e300-01e4-11e9-990b-26eb39efc8f7.png">

We can change the CSS to show it as normal text but even in that way I find it weird.

Reply:
To me, ideally, we should make the whole card the link, similarly to https://console.cloud.google.com/marketplace but I am aware that we can not do it because we have a link to the repository. 

Taking into account that the description is most of the real state of the content of the card, I would make it part of the link too. If the issue is how it looks I would just change it via CSS as you suggested.

Reply:
> We can change the CSS to show it as normal text but even in that way I find it weird.

What do you find weird? The link is actually the wrapper div (card content) no?

Reply:
> What do you find weird? The link is actually the wrapper div (card content) no?

Yes but it's weird for me that I need to override the default link CSS style in order to make it look like normal text while it's actually a link.